api/src/main/java/org/openmrs/Allergy.java

```diff
@@ -160,14 +160,14 @@ public void setSeverity(Concept severity) {
 	 * @return Returns the comment
 	 */
 	public String getComment() {
-		return comment;
+		return comments;
 	}
 	
 	/**
 	 * @param comment the comment to set
 	 */
-	public void setComment(String comment) {
-		this.comment = comment;
+	public void setComment(String comments) {
```

Comment:
You need to deprecate getComment and setComment, add the new getComments and setComments and make sure the old getter/setter delegate to the new ones.

Reply:
In theory you don't even have to change comment to comments for the old getter and setter